docker/entrypoint.sh

```diff
@@ -2,20 +2,20 @@
 
 set -e
 user=ipfs
-repo="$IPFS_PATH"
 
 if [ `id -u` -eq 0 ]; then
+    echo "Changing user to $user"
     # ensure directories are writable
-    su-exec "$user" test -w "$repo" || chown -R -- "$user" "$repo"
-    su-exec "$user" test -w "$IPFS_CLUSTER_PATH" || chown -R -- "$user" "$IPFS_CLUSTER_PATH"
+    su-exec "$user" test -w "${IPFS_PATH}" || chown -R -- "$user" "${IPFS_PATH}"
+    su-exec "$user" test -w "${IPFS_CLUSTER_PATH}" || chown -R -- "$user" "${IPFS_CLUSTER_PATH}"
```

Comment:
Out of curiousity, why add the braces?

Reply:
Just good practice. Ensures no silly errors happen: `"$MY_PATHdefault"` would be substituted for `""` even if your code defines `MY_PATH`. Solution: `"${MY_PATH}default"`.